### PR TITLE
homework 3

### PR DESCRIPTION
**Pull request by artsiom-tsaryonau**

### CODE REVIEW

File: homework3/src/main/java/com/epam/lab/mentoring/console/ConsoleUtils.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+public final class ConsoleUtils {
+
+    public static boolean validateInput(String input) {
+        if (StringUtils.isBlank(input)) {
+            System.out.println("Incorrect input!");
+            return false;
+        }
+        return true;
+    }
+
```
Comment (AShynkevich) [inline homework3/src/main/java/com/epam/lab/mentoring/console/ConsoleUtils.java]: Don't forget to remove extra lines

File: homework3/src/main/java/com/epam/lab/mentoring/watcher/WatcherService.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+import java.util.concurrent.ConcurrentLinkedDeque;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+import static java.nio.file.StandardWatchEventKinds.*;
+
+class WatcherService {
```
Comment (AShynkevich) [inline homework3/src/main/java/com/epam/lab/mentoring/watcher/WatcherService.java]: Why is it not public?

File: homework3/src/main/java/com/epam/lab/mentoring/console/ConsoleUtils.java (hunk 1 above)
Comment (artsiom-tsaryonau) [reply homework3/src/main/java/com/epam/lab/mentoring/console/ConsoleUtils.java]: fixed

File: homework3/src/main/java/com/epam/lab/mentoring/watcher/WatcherService.java (hunk 2 above)
Comment (artsiom-tsaryonau) [reply homework3/src/main/java/com/epam/lab/mentoring/watcher/WatcherService.java]: added modifier

**Pull request merged**